checkstyle/suppressions.xml

```diff
@@ -41,6 +41,8 @@
     <suppress checks="NPathComplexity" files="(ClusterTestExtensions|KafkaApisBuilder).java"/>
     <suppress checks="MethodLength"
               files="(KafkaClusterTestKit).java"/>
+    <suppress checks="ImportControl"
+              files="(ZkBrokerRegistrationTest|DelegatingRequest|InstrumentedRequestProcessor|InstrumentedZooKeeperServer|ReceiptEvent|ZkBrokerRegistrationStubs).java"/>
```

Comment:
Apologies if I am missing the obvious, but am I getting it correctly that it is not only an ordering problem, but that the packages of the imported classes need to be defined in [import-control-core.xml](https://github.com/apache/kafka/blob/trunk/checkstyle/import-control-core.xml)? I will modify that file and remove the suppressions rule.

Reply:
Added the packages to import-control-core in [this commit](https://github.com/apache/kafka/pull/13558/commits/03d444b94de2494548b61cd0458c842c05872e3f). I had to kept a suppression rule for the package `org.apache.zookeeper` because it is not part of any import control and Checkstyle would not accept it without it.